runtime/parser/declaration.go

```diff
@@ -214,7 +214,10 @@ func parseDeclaration(p *parser, docString string) (ast.Declaration, error) {
 				purity = parsePurityAnnotation(p)
 				continue
 
-			case KeywordPriv, KeywordPub, KeywordAccess:
+			case KeywordPub, KeywordPriv:
+				return nil, p.syntaxError(fmt.Sprintf("`%s` is no longer a valid access keyword", p.currentTokenSource()))
```

Comment:
👌

Reply:
Maybe in a follow up PR we could make this a dedicated error and suggest the appropriate replacement in the secondary message